pages/api/journal/deleteById.ts

```diff
@@ -0,0 +1,17 @@
+import { NextApiResponse, NextApiRequest } from "next";
+import {deleteJournalEntryById} from "server/actions/Contentful";
+
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse){
+
+    if(req.method == "DELETE"){
+        if (!req.query.id) res.status(400).json({message:"Bad request."});
```

Comment:
might want to use `res.end()` after setting status `400` but im not 100% sure if its needed